src/Test.UnitTests.Sarif/TestData/RoundTripping/RoundTripping.sarif

```diff
@@ -26,7 +26,7 @@
             "dateTimeArray": [
   "2019-09-26T15:52:02",
   "2019-09-26T15:54",
-  "2019-09-26T15:55:05.123456789"
+  "2019-09-26T15:55:05.1234567"
```

Comment:
>1234567 [](start = 23, length = 7)

Now that we once again rely on Newtonsoft.Json's serialization and deserialization of `DateTime`s, they only round-trip 7 fractional second digits.

Reply:
That seems sufficient.